tracer/src/Datadog.Trace/Iast/Aspects/MongoDB/MongoDatabaseAspect.cs

```diff
@@ -53,7 +53,7 @@ public class MongoDatabaseAspect
         }
         catch (global::System.Exception ex)
         {
-            IastModule.Log.Warning(ex, $"Error invoking {nameof(MongoDatabaseAspect)}.{nameof(AnalyzeCommand)}");
+            IastModule.Log.Error(ex, $"Error invoking {nameof(MongoDatabaseAspect)}.{nameof(AnalyzeCommand)}");
```

Comment:
I'd rather add a `LogAspectException` function in the `IastModule` and call it, so if we want to change the functionality in the future we will only have to touch that function instead of all call sites

Reply:
Also, it can ensure no Error flooding happens, sending only one of each

Reply:
Good idea, I'll do it in another PR 😄